sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -692,14 +696,14 @@ def _send_message_impl(self, message, **kwargs):
                     condition=ErrorCondition.UnknownError, description="Send failed."
                 ) from None
 
-    def send_message(self, message, **kwargs):
+    def send_message(self, message, *, timeout: float = 0, **kwargs): # pylint:disable=unused-argument
         """
         :param ~pyamqp.message.Message message:
         :keyword float timeout: timeout in seconds. If set to
          0, the client will continue to wait until the message is sent or error happens. The
          default is 0.
         """
-        self._do_retryable_operation(self._send_message_impl, message=message, **kwargs)
+        self._do_retryable_operation(self._send_message_impl, message=message, timeout=timeout)
```

Comment:
kwargs no longer being passed through, there aren't any that we may want to pass through?